src/controls.css

```diff
@@ -0,0 +1,40 @@
+.controls-title {
+  font-size: 40px;
+  margin-bottom: 20px;
+}
+
+.controls i{
+  color: cadetblue;
+  margin: 0 0.6rem;
+}
+
+.controls i:hover{
+  color: cornflowerblue;
+}
+
+.controls ol{
+  padding-left: 10px;
+}
+
+.controls li{
+  display: flex;
+  align-items: center;
+  font-size: 20px;
+}
+
+p{
```

Comment:
This selects all the paragraph tag on the page. Give this `p` a className and then use that className to add the CSS rules.